### ISSUE TITLE
4.x Detect scripts with trailing slash in URL

### ISSUE THREAD

**Issue by chelobaka** [greasemonkey/greasemonkey#2642]
Version 4.0 doesn't seem to detect user scripts if URL has a trailing slash, like: `https://www.company.com/scripts/fancy.user.js/`. It worked in 3.x versions.

**Comment by Sxderp** [comment]
Probably just need to update the matches in the manifest. What I find my interesting is that you came across a web host serving files at what _should_ be a directory listing.

**Comment by chelobaka** [comment]
It is an internal company hosting and since scripts are intended for internal use only I can't host them on the Internet.

**Comment by arantius** [comment]
I've just installed 3.17 and used the [user script server](https://bugzilla.mozilla.org/attachment.cgi?id=8922013&action=edit) I wrote for something else (which serves a simple user script at every URL).

When I open `/foo.user.js` I get an install dialog.  When I open `/foo.user.js/` I do not.  I don't believe this ever worked, I know it was never intended to work, and I don't plan on making it work in 4.x.
